<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Daftar Persil Yang Bisa Diedit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="table_daftar_persil">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Info Berkas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree_info_berkas">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Editor Persil</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Nama Jalan</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="input_nama_jalan"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Nomor</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="input_nomor"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Kode Pos</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_kode_pos"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Alamat Tambahan</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="input_alamat_tambahan"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Penggunaan Umum</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="combo_penggunaan_umum"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Penggunaan Khusus</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="combo_penggunaan_khusus"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Peta</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="input_peta"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Nomor Peta (Lokal)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="input_nomor_peta_lokal"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Nomor Lembar (TM3)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="input_nomor_lembar"/>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Nomor Kotak</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_nomor_kotak"/>
           </item>
          </layout>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Keadaan Tanah</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QTextEdit" name="input_keadaan_tanah"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Tanda Batas</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QTextEdit" name="input_tanda_batas"/>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Penunjuk Batas</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QTextEdit" name="input_penunjuk_batas"/>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Hal lain-lain</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QTextEdit" name="input_hal_lain_lain"/>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Metode Ukur</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QComboBox" name="combo_metode_ukur"/>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Alat Ukur</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QComboBox" name="combo_alat_ukur"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="btn_simpan">
           <property name="text">
            <string>Simpan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_tutup">
           <property name="text">
            <string>Tutup</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
